enterprise/backend/src/metabase_enterprise/ai_entity_analysis/api.clj

```diff
@@ -0,0 +1,66 @@
+(ns metabase-enterprise.ai-entity-analysis.api
+  "`/api/ee/ai-entity-analysis/` routes"
+  (:require
+   [metabase-enterprise.metabot-v3.client :as metabot-client]
+   [metabase.api.macros :as api.macros]
+   [metabase.api.routes.common :refer [+auth]]
+   [metabase.premium-features.core :as premium-features]
+   [metabase.util.i18n :refer [tru]]
+   [metabase.util.malli.schema :as ms]))
+
+(set! *warn-on-reflection* true)
+
+(api.macros/defendpoint :post "/analyze-chart"
+  "Analyze a chart image using an AI vision model."
+  [_route-params
+   {model :model
+    focus-instructions :focus_instructions
+    extra-context :extra_context
```

Comment:
These seem to be unused. Unused variables should start with `_` (or removed), otherwise the linter starts shouting. If we are not sure if and how these will be used in the near future, I'd delete them.